### PR TITLE
Add mul absorbing for smooth quant of onnxrt adaptor

### PR DESCRIPTION
**Pull request by mengniwang95**
## Type of Change

feature

## Description

Add mul absorbing function for smooth quant of onnxrt adaptor

## Expected Behavior & Potential Risk

UT pass

## How has this PR been tested?

UT, local LLM test

## Dependency Change?

no

### CODE REVIEW

**Comment by mengniwang95** [comment]
test 2 nlp models with sq

**Comment by mengniwang95** [comment]
> test 2 nlp models with sq https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4888/

https://inteltf-jenk.sh.intel.com/job/intel-lpot-validation-top-mr-extension/4889/

File: neural_compressor/adaptor/onnxrt.py [hunk 1]
```diff
@@ -233,6 +237,9 @@ def smooth_quant(self, model, dataloader, iterations, tune_cfg, alpha=0.5, foldi
         self.pre_optimized_model.update()
         self.pre_optimized_model.topological_sort()
         self.pre_optimized_model.remove_unused_constant()
+
+        absorb_scale(self.pre_optimized_model, scales)
```
Comment (hshen14) [inline neural_compressor/adaptor/onnxrt.py:241]: can we use fold_scale, which aligns PT usage？

**Comment by chensuyue** [comment]
[extension test](https://inteltf-jenk.sh.intel.com/job/inc-validation-smooth_quant-top-weekly/3/)

**Comment by chensuyue** [comment]
SmoothQuant Vs Common Quant [report](https://inteltf-jenk.sh.intel.com/view/lpot-validation/job/intel-lpot-generate_report/186/artifact/report.html)

**Review by wenhuach21 — APPROVED** [review]

**Pull request merged**